experiments/standalone/Bucket/global_bucket_function.jl

```diff
@@ -164,6 +169,38 @@ updatefunc = ClimaLand.make_update_drivers(bucket_atmos, bucket_rad)
 driver_cb = ClimaLand.DriverUpdateCallback(updateat, updatefunc)
 cb = SciMLBase.CallbackSet(driver_cb, saving_cb)
 
+
+
+# ClimaDiagnostics
+output_dir = "output/"
```

Comment:
Let's use a more descriptive name, maybe `global_bucket_function`? 

This will help when we add the diagnostics to more experiments